packages/mui-lab/package.json

```diff
@@ -64,7 +64,7 @@
     "@mui/base": "5.0.0-beta.11",
     "@mui/system": "^5.14.5",
     "@mui/types": "^7.2.4",
-    "@mui/utils": "^5.14.4",
+    "@mui/utils": "^5.14.5",
```

Comment:
Interesting - I wonder why it hasn't been picked up by lerna when publishing a new version.